### PR TITLE
[Utilities] Renamed Deployment History Removal functions & fixed reference

### PR DESCRIPTION
**Pull request by AlexanderSehr**
# Description

- Renamed Deployment History Removal functions
- Fixed pipeline reference in GitHub pipeline
- Removed duplicated docs

| Pipeline |
| - |
| [Build Status](https://dev.azure.com/carml/CARML/_build/latest?definitionId=124&branchName=users%2Falsehr%2FworkflowRename) |

# Type of Change

Please delete options that are not relevant.

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Update to documentation

### CODE REVIEW

File: .github/workflows/platform.deployment.history.removal.yml [hunk 1]
```diff
@@ -15,15 +15,15 @@ on:
...
+        description: 'The number of days to keep deployments with status [failed]' # 'Running' are always excluded
         required: false
         default: '14'
   schedule:
     - cron: '0 0 * * *'
 
 env:
   variablesPath: 'settings.yml'
-  workflowPath: '.github/workflows/platform.cleanup'
+  workflowPath: 'platform.deployment.history.removal.yml'
```
Comment (eriqua) [inline .github/workflows/platform.deployment.history.removal.yml:26]: Can we really remove `.github/workflows/`?

File: .azuredevops/platformPipelines/platform.deployment.history.removal.yml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-name: '.Platform - Remove Deployments'
+name: '.Platform - Remove Deployment History'
```
Comment (eriqua) [inline .azuredevops/platformPipelines/platform.deployment.history.removal.yml:1]: What about
```suggestion
name: '.Platform - Clean Up Deployment History'
```

File: .github/workflows/platform.deployment.history.removal.yml [hunk 3]
```diff
@@ -1,4 +1,4 @@
-name: '.Platform: Remove Deployments'
+name: '.Platform: Remove Deployment History'
```
Comment (eriqua) [inline .github/workflows/platform.deployment.history.removal.yml:1]: Same as above, plus we should rename the 2 files

Comment (AlexanderSehr) [reply .github/workflows/platform.deployment.history.removal.yml]: Renamed

File: .azuredevops/platformPipelines/platform.deployment.history.removal.yml [hunk 4]
```diff
@@ -0,0 +1,125 @@
+name: '.Platform - Clean Up Deployment History'
```
Comment (eriqua) [inline .azuredevops/platformPipelines/platform.deployment.history.removal.yml:1]: This file should be renamed as well

Comment (eriqua) [inline .azuredevops/platformPipelines/platform.deployment.history.removal.yml:1]: This is probably duplicated?

File: .github/workflows/platform.deployment.history.cleanup.yml [hunk 5]
```diff
@@ -15,15 +15,15 @@ on:
...
+        description: 'The number of days to keep deployments with status [failed]' # 'Running' are always excluded
         required: false
         default: '14'
   schedule:
     - cron: '0 0 * * *'
 
 env:
   variablesPath: 'settings.yml'
-  workflowPath: '.github/workflows/platform.cleanup'
+  workflowPath: '.github/workflows/platform.deployment.history.removal.yml'
```
Comment (eriqua) [inline .github/workflows/platform.deployment.history.cleanup.yml:26]:
```suggestion
  workflowPath: '.github/workflows/platform.deployment.history.cleanup.yml'
```

File: .azuredevops/platformPipelines/platform.deployment.history.removal.yml (hunk 4 above)
Comment (AlexanderSehr) [reply .azuredevops/platformPipelines/platform.deployment.history.removal.yml]: Appears so. Removed

**Review by eriqua — APPROVED** [review]
🆗

**Pull request merged**